wgpu-types/src/lib.rs

```diff
@@ -1731,9 +1703,6 @@ pub struct BindGroupLayoutEntry {
     ///
     /// If this value is Some and `ty` is any other variant, bind group creation will fail.
     pub count: Option<u32>,
-    /// This struct should be partially initalized using the default method, but binding, visibility,
-    /// and ty should be set.
-    pub _non_exhaustive: NonExhaustive,
```

Comment:
The advantage is that this field no longer has to exist in the public API. And we can enforce the fact that `binding, visibility and ty should be set` because people are now forced to use the `new()` method.